libs/tools/builders/elements/build.ts

```diff
@@ -0,0 +1,155 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BuilderContext, BuilderOutput } from '@angular-devkit/architect';
+import { promises as fs } from 'fs';
+import { join } from 'path';
+import { forkJoin, from, Observable, of } from 'rxjs';
+import { catchError, map, mapTo, switchMap, tap } from 'rxjs/operators';
+import { NxJson, PackageJson, tryJsonParse } from '../util/json-utils';
+import { ElementsOptions } from './schema';
+
+/**
+ * -----------------------------------------------------------
+ */
+export function elementsBuilder(
+  options: ElementsOptions,
+  context: BuilderContext,
+): Observable<BuilderOutput> {
+  const project = context.target !== undefined ? context.target!.project : '';
+  context.logger.info(`Packaging ${project}...`);
+
+  // Get all builds that are tagged with elements
+  context.logger.info('Reading nx.json file...');
+  return from(
+    tryJsonParse<NxJson>(join(context.workspaceRoot, 'nx.json')),
+  ).pipe(
+    // Find the projects that need to be built as part of the elements build.
+    map((nxJson: NxJson) =>
+      filterTaggedProjects(nxJson, options.buildTag || 'scope:elements'),
+    ),
+    // Create and schedule all other builds.
+    switchMap((targetProjects: string[]) => {
+      const targets = createProjectBuildStreams(context, targetProjects);
+      return forkJoin(...targets);
+    }),
+    switchMap(() =>
+      // Copy the root package json and sync dependency versions.
+      copyRootPackageJson(
+        context,
+        options.releasePackageJson,
+        options.outputPath,
+      ),
+    ),
+    tap(() => context.logger.info('Build successful.')),
+    // When the builds have finished, create the root package json
+    // and return the results.
+    mapTo({
+      success: true,
+    }),
+    catchError(error => {
+      return of({
+        success: false,
+        error: (error as Error).message,
+      });
+    }),
+  );
+}
+
+/** Filter incoming projects based on the tag defined in the options. */
+function filterTaggedProjects(nxJson: NxJson, buildTag: string): string[] {
+  return Object.entries(nxJson.projects)
+    .filter(([_project, { tags }]) => tags && tags.includes(buildTag))
```

Comment:
add filtering for your own name. You should get your own name via the `BuilderContext`.